docs/intro/tutorial.rst

```diff
@@ -39,6 +41,7 @@ as well as the `suggested resources in the learnpython-subreddit`_.
 .. _Python: https://www.python.org/
 .. _this list of Python resources for non-programmers: https://wiki.python.org/moin/BeginnersGuide/NonProgrammers
 .. _Python Tutorial: https://docs.python.org/3/tutorial
+.. _a jupyter notebook: https://github.com/scrapy/scrapy/blob/master/docs/intro/Scrapy_Tutorial.ipynb
```

Comment:
I meant the kind of link you used the first time, where users will be able to use the notebook online directly by just following the link.